### PR TITLE
[percona] update psmdb-36 version

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: library/percona [hunk 1]
```diff
@@ -30,7 +30,7 @@ GitCommit: fc7952f19d7e3e55477a9baee2f6d4d1092431f6
 Directory: percona-server-mongodb.40
 File: Dockerfile
 
-Tags: psmdb-3.6.16, psmdb-3.6
-GitCommit: fc7952f19d7e3e55477a9baee2f6d4d1092431f6
+Tags: psmdb-3.6.17 psmdb-3.6
```
Comment (yosifkit) [inline library/percona]:
```suggestion
Tags: psmdb-3.6.17, psmdb-3.6
```

Comment (EvgeniyPatlan) [reply library/percona]: fixed

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index d9f317a..644fea1 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -17,7 +17,7 @@ percona:8.0.18-9-centos
 percona:centos
 percona:latest
 percona:psmdb-3.6
-percona:psmdb-3.6.16
+percona:psmdb-3.6.17
 percona:psmdb-4.0
 percona:psmdb-4.0.14
 percona:psmdb-4.2
diff --git a/percona_psmdb-3.6/Dockerfile b/percona_psmdb-3.6/Dockerfile
index 05f014f..0215747 100644
--- a/percona_psmdb-3.6/Dockerfile
+++ b/percona_psmdb-3.6/Dockerfile
@@ -18,7 +18,7 @@ plus additional enterprise-grade functionality."
 LABEL org.opencontainers.image.license="SSPLv1"
 LABEL org.opencontainers.image.authors="info@percona.com"
 
-ENV PSMDB_VERSION 3.6.16-3.6
+ENV PSMDB_VERSION 3.6.17-4.0
 LABEL org.label-schema.schema-version=${PSMDB_VERSION}
 LABEL org.opencontainers.image.version=${PSMDB_VERSION}
```

</details>

**Comment by tianon** [comment]
LGTM

Build test of #7466; 44f3d013; `amd64` (`percona:psmdb-3.6.17`):

```console

$ bashbrew build percona:psmdb-3.6.17
Building bashbrew/cache:14290a17b11c192bfe798376bdac7815d221e3d1acccff3c88a0f8c7c10c59b5 (percona:psmdb-3.6.17)
Tagging percona:psmdb-3.6.17
Tagging percona:psmdb-3.6

$ test/run.sh percona:psmdb-3.6.17
testing percona:psmdb-3.6.17
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/8]...+ exec echo 'Hello world-20545-3191'
passed
	'mongo-basics' [5/8]....passed
	'mongo-auth-basics' [6/8].....passed
	'mongo-tls-basics' [7/8]....passed
	'mongo-tls-auth' [8/8]....passed

```

**Pull request merged**